src/components/Article.jsx

```diff
@@ -1 +1,15 @@
-// export function Article() {}
+import React from 'react';
+
+export function Article({
+  tittle,
+  text,
+  date,
+}) {
+  return (
+    <>
+      <h1>{ tittle }</h1>
```

Comment:
I think you don't need these indents.
```suggestion
      <h1>{tittle}</h1>
```